.github/workflows/tf-account.yaml

```diff
@@ -12,10 +12,17 @@ on:
         required: true
       TF_BACKEND_S3_REGION:
         required: true
+    inputs:
+      TERRAFORM_VERSION:
```

Comment:
Workflow inputs are very similar to Actions inputs and should use the same conventions:

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

Meaning this is either `terraform-version` or `terraform_version`. Both are valid, but dash case is canonical.

In an action, inputs are passed through via upper-cased environment variables and are thus case insensitive. In workflow calls, they probably are case sensitive.

Secrets are uppercased, following environment variable conventions.